Makefile.am

```diff
@@ -39,7 +39,7 @@ dist_doc_DATA = \
 	COPYING
 
 AM_CPPFLAGS = $(OPENSSL_CRYPTO_CFLAGS) -D_UNICODE
-AM_CFLAGS = -municode
+AM_CFLAGS = -municode -Werror
```

Comment:
We have -Wall and -Wextra turned on, adding -Werror to that could become very limiting. The code wont compile until all warnings are eliminated is not where I would like to go... Let's see what others say.